recipe/meta.yaml

```
@@ -1,39 +1,39 @@
 {% set name = "ydata-profiling" %}
-{% set version = "4.8.3" %}
+{% set version = "4.16.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ydata-profiling-{{ version }}.tar.gz
-  sha256: d9467ecc8474344347ad89a6839db14762f0230e53df94f83a04810b2fa6cbd7
+  sha256: 4aa0d6412a140194396e34f5ffd348223f0319d877b0243e765f30ae633ed51e
 
 build:
+  number: 0
   entry_points:
     - ydata_profiling = ydata_profiling.controller.console:main
     - pandas_profiling = ydata_profiling.controller.console:main
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
-  number: 0
-  # numba is not available on s390x
-  skip: True   #  [py<37 or s390x]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<37 or py>312]
 
 requirements:
   host:
     - python
     - pip
-    - setuptools
-    - wheel
+    - setuptools >=72.0.0,<80.0.0
+    - setuptools-scm >=8.0.0,<9.0.0
+    - wheel >=0.38.4,<1.0.0
   run:
     - python
-    - scipy  >=1.4.1,<1.14
+    - scipy >=1.4.1,<1.16
     - pandas >1.1,<3,!=1.4.0
-    - matplotlib-base >=3.2,<3.9
+    - matplotlib-base >=3.5,<3.10
     - pydantic >=2
     - pyyaml >=5.0.0,<6.1
     - jinja2 >=2.11.1,<3.2
-    - visions >=0.7.5,<0.7.7
-    - numpy >=1.16.0,<2
+    - visions >=0.7.5,<0.8.2
```

```suggestion
    - visions >=0.7.5,<0.8.2
    - imagehash
    - pillow
```
because of https://github.com/dylan-profiler/visions/blob/v0.8.1/pyproject.toml#L38